### PR TITLE
chore(package): downgrade eslint, 6.2 causes failures

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1727,7 +1727,7 @@
         },
         "util": {
           "version": "0.10.3",
-          "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+          "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```
Comment (sidvishnoi) [inline package-lock.json]:
```suggestion
          "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
```

File: package-lock.json [hunk 2]
```diff
@@ -8730,7 +8724,7 @@
     },
     "sha.js": {
       "version": "2.4.11",
-      "resolved": "https://registry.npmjs.org/sha.js/-/sha.js-2.4.11.tgz",
+      "resolved": "http://registry.npmjs.org/sha.js/-/sha.js-2.4.11.tgz",
```
Comment (sidvishnoi) [inline package-lock.json]:
```suggestion
      "resolved": "https://registry.npmjs.org/sha.js/-/sha.js-2.4.11.tgz",
```

File: package-lock.json (hunk 1 above)
Comment (marcoscaceres) [reply package-lock.json]: just let it do its thing...

File: package-lock.json [hunk 3]
```diff
@@ -8751,7 +8745,7 @@
       "dependencies": {
         "kind-of": {
           "version": "2.0.1",
-          "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
+          "resolved": "http://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
```
Comment (sidvishnoi) [inline package-lock.json]:
```suggestion
          "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
```

File: package-lock.json [hunk 4]
```diff
@@ -10136,7 +10130,7 @@
     },
     "tty-browserify": {
       "version": "0.0.0",
-      "resolved": "https://registry.npmjs.org/tty-browserify/-/tty-browserify-0.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/tty-browserify/-/tty-browserify-0.0.0.tgz",
```
Comment (sidvishnoi) [inline package-lock.json]:
```suggestion
      "resolved": "https://registry.npmjs.org/tty-browserify/-/tty-browserify-0.0.0.tgz",
```

File: package-lock.json [hunk 5]
```diff
@@ -10875,7 +10869,7 @@
     },
     "xmlbuilder": {
       "version": "9.0.7",
-      "resolved": "https://registry.npmjs.org/xmlbuilder/-/xmlbuilder-9.0.7.tgz",
+      "resolved": "http://registry.npmjs.org/xmlbuilder/-/xmlbuilder-9.0.7.tgz",
```
Comment (sidvishnoi) [inline package-lock.json]:
```suggestion
      "resolved": "https://registry.npmjs.org/xmlbuilder/-/xmlbuilder-9.0.7.tgz",
```

**Pull request merged**